tests/testthat/test-stacking.R

```diff
@@ -728,3 +728,22 @@ if (file.exists("cola19 stacked.sav"))
 
 if (file.exists("cola20 stacked.sav"))
     file.remove("cola20 stacked.sav")
+
+test_that("DS-4405: Stacking with duplicate labels",
+{
+    expect_equal(deduplicateValAttrNames(list(Dislike = 0, Dislike = 1, Hate = 2)),
+                 list("Dislike [Value: 0]" = 0, "Dislike [Value: 1]" = 1, Hate = 2))
+    input.file <- findInstDirFile("testdata/ColaTracking-bugdata.sav")
```

Comment:
"testdata" not needed here I think. Only works locally when I remove it and just have the file name.

Reply:
Makes sense now seeing the code.